### PR TITLE
Jp 411 pub

### PR DESCRIPTION
**Pull request by jaybdub**
- add automatic memory configuration based on platform (by checking available RAM)
- change jetbot remotes for 32.4.4 to docker hub
- remove docker configuration steps from software setup/docker

### CODE REVIEW

File: docs/software_setup/docker.md [hunk 1]
```diff
@@ -50,34 +50,11 @@ cd jetbot
...
-export JETBOT_JUPYTER_MEMORY_SWAP=3G
-```
-
-
 ## Step 5 - Enable all containers
 
 Call the following to enable the JetBot docker containers 
 
 ```bash
-sudo systemctl enable docker   # enable docker daemon at boot
```
Comment (tokk-nv) [inline docs/software_setup/docker.md:80]: so, do we figure out that we don't need to manually enable docker service?

File: docs/software_setup/docker.md [hunk 2]
```diff
@@ -50,34 +50,11 @@ cd jetbot
...
 ./enable.sh $HOME   # we'll use home directory as working directory, set this as you please.
```
Comment (tokk-nv) [inline docs/software_setup/docker.md:58]: we want to put `cd docker` before?

File: docs/software_setup/docker.md (hunk 1 above)
Comment (jaybdub) [reply docs/software_setup/docker.md:80]: Good catch, We still need it.  Resolved here, https://github.com/NVIDIA-AI-IOT/jetbot/pull/294/commits/4c20635ac693191da87fbeffa3d04939c85f763b

**Review by tokk-nv — APPROVED** [review]
These now look good.

**Pull request merged**